<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameDetailWidget</class>
 <widget class="QWidget" name="GameDetailWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1048</width>
    <height>1087</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>游戏详情</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: linear-gradient(135deg, #f5f7fa 0%, #c3cfe2 100%);
    font-family: &quot;Microsoft YaHei&quot;, sans-serif;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="coverwidget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>300</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>350</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background: white;
    border-radius: 12px;
    margin: 12px;
    padding: 20px;
    border: none;
    box-shadow: 0 4px 12px rgba(0, 0, 0, 0.1);</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="titlelabel">
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>20</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">font-family: &quot;微软雅黑&quot;;
    font-size: 20pt;
    font-weight: bold;
    color: #1a237e;
    text-align: center;
    padding: 15px 0;
    border-bottom: 2px solid #e8eaf6;
    margin-bottom: 15px;</string>
        </property>
        <property name="text">
         <string>游戏标题</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>300</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background: #f8f9fa;
    border: 1px solid #e0e0e0;
    border-radius: 8px;
    padding: 12px;
    font-size: 10.5pt;
    line-height: 160%;
    color: #424242;
    font-family: &quot;Microsoft YaHei&quot;, sans-serif;
box-shadow: 0 0 0 2px rgba(63, 81, 181, 0.1);
    transition: all 0.3s ease;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="styleSheet">
      <string notr="true">background: white;
    border-radius: 12px;
    margin: 12px;
    padding: 20px;
    border: none;
    box-shadow: 0 4px 12px rgba(0, 0, 0, 0.1);</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>Microsoft YaHei</family>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size: 11pt;
    font-weight: bold;
    color: #303f9f;
    padding: 8px 0 4px 0;
    border-bottom: 1px solid #e8eaf6;
    margin-bottom: 8px;</string>
        </property>
        <property name="text">
         <string>游戏信息</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="textBrowser">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>150</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>200</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background: #f8f9fa;
    border: 1px solid #e0e0e0;
    border-radius: 8px;
    padding: 12px;
    font-size: 10.5pt;
    line-height: 160%;
    color: #424242;
    font-family: &quot;Microsoft YaHei&quot;, sans-serif;
box-shadow: 0 0 0 2px rgba(63, 81, 181, 0.1);
    transition: all 0.3s ease;</string>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Microsoft YaHei','sans-serif'; font-size:10.5pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'SimSun'; font-size:9pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <family>Microsoft YaHei</family>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size: 11pt;
    font-weight: bold;
    color: #303f9f;
    padding: 8px 0 4px 0;
    border-bottom: 1px solid #e8eaf6;
    margin-bottom: 8px;</string>
        </property>
        <property name="text">
         <string>游戏奖项</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>Microsoft YaHei</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size: 9.5pt;
    color: #546e7a;
    line-height: 160%;
    padding: 8px;
    background: #f8f9fa;
    border-radius: 6px;
    border-left: 4px solid #3f51b5;
    margin: 5px 0;</string>
        </property>
        <property name="text">
         <string>暂无奖项信息</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
